lib/matplotlib/axis.py

```diff
@@ -1023,6 +1023,76 @@ def set_default_intervals(self):
         # attribute, and the derived code below will check for that
         # and use it if it's available (else just use 0..1)
 
+    def _set_lim(self, v0, v1, *, emit=True, auto):
+        """
+        Set view limits.
+
+        This method is a helper for the Axes ``set_xlim``, ``set_ylim``, and
+        ``set_zlim`` methods.  This docstring uses names corresponding to
+        ``set_xlim`` for simplicity.
+
+        *v0* and *v1* are the view limits; they must be two scalars (passing a
+        *v0* as a (low, high) pair is not supported).  *emit* and *auto* are
+        forwarded as-is from the Axes limits setter.
```

Comment:
I think it's better to have a standard parameters section,  and not write the docstring with the perspective of the calling functions in mind.

```suggestion
        Parameters
        ----------
        v0, v1 : float
            The view limits.
        emit : bool, default: True
            Whether to notify observers of limit change.
        auto : bool or None
            Whether to turn on autoscaling of the x-axis. True turns on,
            False turns off, None leaves unchanged.
```

Reply:
sure